### PR TITLE
feat: Display info from new COA checkboxes to user on artwork page

### PR DESCRIPTION
**Pull request by annacarey**
Added new copy about certificate of authentication to provide more detail around COA—whether the COA was issued by an authorized authenticating body and/or by the gallery. These correspond to new checkbox inputs in the artwork form in CMS that allows galleries to provide this further detail.

Case for w/ issued by gallery checked:
Screen Shot 2021-02-02 at 1 34 55 PM
Case for w/ issued by authenticating body checked:
Screen Shot 2021-02-02 at 1 29 24 PM
Case for w/ both checked:
Screen Shot 2021-02-03 at 2 36 03 PM

[Jira ticket](https://artsyproduct.atlassian.net/browse/PX-3608)

### CODE REVIEW

File: src/schema/v2/artwork/__tests__/artwork.test.js
Comment (sweir27) [inline src/schema/v2/artwork/__tests__/artwork.test.js:2300]: #minor #non-blocking we are totally not consistent about this, but I find test files to be slightly more readable when there's a blank line between subsequent tests/blocks. (Usually when I'm in a file I try to update it!)

Also #superminor typo here `ertificate_of_authenticity`.

**Review by sweir27 — APPROVED** [review]
Seems straightforward 👍 from me!

**Comment by sweir27** [comment]
@annacarey it's usually helpful to include a screenshot of the query/response here. I do trust your tests but it's nice to have the confirmation (and those are usually extra helpful down the line for putting the change in context).

File: src/schema/v2/artwork/__tests__/artwork.test.js
Comment (annacarey) [reply src/schema/v2/artwork/__tests__/artwork.test.js:2300]: OOOH thank you good call on both!

File: src/schema/v2/artwork/__tests__/artwork.test.js
Comment (oxaudo) [inline src/schema/v2/artwork/__tests__/artwork.test.js:2315]: #superminor: probably should be `to proper object when`....

File: src/schema/v2/artwork/__tests__/artwork.test.js
Comment (oxaudo) [inline src/schema/v2/artwork/__tests__/artwork.test.js:2327]: #superminor: copy paste from above -:)

**Review by oxaudo — APPROVED** [review]
Had one super minor note just to say something:)

Please merge when you get a clear from Arielle.

File: src/schema/v2/artwork/__tests__/artwork.test.js
Comment (annacarey) [reply src/schema/v2/artwork/__tests__/artwork.test.js]: thanks for catching that and taking a look. fixed! just waiting on final A-OK from Arielle and then will merge!

**Pull request merged**
